packages/ibm-products-styles/src/components/SidePanel/_side-panel.scss

```
@@ -109,7 +109,7 @@ $action-set-block-class: #{c4p-settings.$pkg-prefix}--action-set;
     left: 0;
     border-right: 1px solid $border-subtle-02;
   }
-  &.#{$block-class}.#{$block-class}--has-slug {
+  &.#{$block-class}.#{$block-class}--has-ai-label {
```

I believe the decision was to include _duplicate_ selectors[^1] to be backwards compatible.

```suggestion
  &.#{$block-class}.#{$block-class}--has-slug,
  &.#{$block-class}.#{$block-class}--has-ai-label {
```

[^1]: https://github.com/carbon-design-system/carbon/issues/17714